test/unit/reports/get-report-lines.test.js

```diff
@@ -0,0 +1,131 @@
+const { TRANSACTION } = require('../../../app/constants/transaction')
+const { FRN } = require('../../mocks/values/frn')
+const { UNKNOWN } = require('../../../app/constants/unknown')
+const { INVOICE_NUMBER } = require('../../mocks/values/invoice-number')
+const { CURRENCY } = require('../../mocks/values/currency')
+const { VALUE } = require('../../mocks/values/value')
+const { DATE } = require('../../mocks/values/date')
+const { PAYMENT_ENRICHED_STATUS } = require('../../../app/constants/statuses')
+const { CORRELATION_ID } = require('../../mocks/values/correlation-id')
+
+jest.mock('../../../app/reports/mi-report/get-frn')
+const { getFrn: mockGetFrn } = require('../../../app/reports/mi-report/get-frn')
+
+jest.mock('../../../app/reports/mi-report/get-invoice-number')
+const { getInvoiceNumber: mockGetInvoiceNumber } = require('../../../app/reports/mi-report/get-invoice-number')
+
+jest.mock('../../../app/reports/mi-report/get-currency')
+const { getCurrency: mockGetCurrency } = require('../../../app/reports/mi-report/get-currency')
+
+jest.mock('../../../app/reports/mi-report/get-value')
+const { getValue: mockGetValue } = require('../../../app/reports/mi-report/get-value')
+
+jest.mock('../../../app/reports/mi-report/get-batch-export-date')
+const { getBatchExportDate: mockGetBatchExportDate } = require('../../../app/reports/mi-report/get-batch-export-date')
+
+jest.mock('../../../app/reports/mi-report/get-status')
+const { getStatus: mockGetStatus } = require('../../../app/reports/mi-report/get-status')
```

Comment:
I would probably put the mocks at the beginning of the test file, unless you think otherwise?

Reply:
Agreed, keeps it consistent with other files